ajax/libs/cachep2p/package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "cachep2p",
+  "filename": "cachep2p.min.js",
+  "description": "The Peer to Peer Cache for the Masses",
+  "version": "0.1.5",
+  "author": {
+    "name": "Carlos Guerrero",
+    "email": "carlos@carlosguerrero.com",
+    "url": "https://cachep2p.com"
+  },
+  "homepage": "https://cachep2p.com",
```

Comment:
yes, I have confirmed that! And here is the issue about the site: guerrerocarlos/CacheP2P#24